LICENSE.txt

```diff
@@ -1,27 +0,0 @@
-Copyright (c) 2015-2021, conda-forge
```

Comment:
This is removing the main license from the staged-recipes repository and should not be touched at all by this PR.